src/client/datascience/types.ts

```diff
@@ -247,13 +244,7 @@ export interface IHoverProvider extends HoverProvider {}
 export const IHoverProvider = Symbol('IHoverProvider');
 
 export const INotebookExecutionLogger = Symbol('INotebookExecutionLogger');
-export interface INotebookExecutionLogger extends IDisposable {
-    preExecute(cell: ICell, silent: boolean): Promise<void>;
-    postExecute(cell: ICell, silent: boolean, language: string, resource: Uri): Promise<void>;
-    nativePostExecute?(cell: NotebookCell): Promise<void>;
-    onKernelRestarted(resource: Uri): void;
-    preHandleIOPub?(msg: KernelMessage.IIOPubMessage): KernelMessage.IIOPubMessage;
-}
+export interface INotebookExecutionLogger extends IDisposable {}
```

Comment:
Can't this just be eliminated now?

Reply:
I wanted to leave that for debt, there are places where this is used.
Was too much work to try and remove its usages (I only remvoed the methods so that we don't accidentally use them, that's kinda what happened, the methods exist, and we're assuming its all ok, but since nothing is wired up, it will not work).